frontend/app-preview/src/views/LandingPage.tsx

```diff
@@ -4,79 +4,112 @@ import { PreviewContext } from '../PreviewContext';
-  const { data: instanceId } = useInstanceIdQuery(org, app);
-  const selectedLayoutInEditor = localStorage.getItem(instanceId);
   const localSelectedViewSize = localStorage.getItem('viewSize');
   const [viewSize, setViewSize] = useState<string>(localSelectedViewSize ?? 'desktop');
   const selectedLayoutSetInEditor = localStorage.getItem('layoutSetName');
-
-
+  const { data: user } = useUserQuery();
   const isIFrame = (input: HTMLElement | null): input is HTMLIFrameElement =>
     input !== null && input.tagName === 'IFRAME';
 
   if (previewConnection) {
-    previewConnection.on("ReceiveMessage", function (message) {
+    previewConnection.on('ReceiveMessage', function (message) {
       const frame = document.getElementById('app-frontend-react-iframe');
       if (isIFrame(frame) && frame.contentWindow) {
         const targetOrigin = window.origin;
         // Trigger a reload of preview window until app-frontend implements re-calling api #https://github.com/Altinn/app-frontend-react/issues/1088
         window.location.reload();
-        console.log("Sending reload message to app-frontend with targetOrigin: " + targetOrigin);
+        console.log('Sending reload message to app-frontend with targetOrigin: ' + targetOrigin);
         frame.contentWindow.postMessage({ action: message }, targetOrigin);
       }
-    })
+    });
   }
 
   const handleChangeViewSizeClick = (selectedViewSize: string) => {
-      localStorage.setItem('viewSize', selectedViewSize);
-      setViewSize(selectedViewSize);
+    localStorage.setItem('viewSize', selectedViewSize);
+    setViewSize(selectedViewSize);
   };
 
   return (
-      <PreviewContext>
+    <PreviewContext>
+      <>
         <div className={classes.header}>
-          <a href={`/editor/${org}/${app}/ui-editor?layout=${selectedLayoutInEditor}`}>
-            <AltinnStudioLogo />
-          </a>
-          <div className={classes.betaTag}>
-            {'BETA'}
-          </div>
+          <AltinnHeader
+            menu={undefined}
+            showSubMenu={showSubMenu}
+            subMenuContent={subMenuContent()}
+            activeMenuSelection={undefined}
+            org={org}
+            app={app}
+            user={user}
+            repository={repository}
+            buttonActions={buttonActions(org, app)}
+            variant={variant}
+          />
         </div>
         <div className={classes.subHeader}>
           <span className={classes.viewSizeButtons}>
-          <ToggleButtonGroup
-            items={[
-              {
-                label: t('preview.view_size_desktop'),
-                value: 'desktop'
-              },
-              {
-                label: t('preview.view_size_mobile'),
-                value: 'mobile'
-              }
-            ]}
-            onChange={handleChangeViewSizeClick}
-            selectedValue={viewSize === 'desktop' ? 'desktop' : 'mobile'}/>
-            </span>
+            <ToggleButtonGroup
```

Comment:
All of this showing in the "submenu" that you have implemented should be defined as the `subMenuContent` and passed to the AltinnHeader, in the same way that is done for app-development.